pr_3/javascript/random_samples/archive_path_overwrite.js

```
@@ -0,0 +1,99 @@
+    stream.pipe(out);
+    stream.on('end', () => {
+        next();
+    })
+    stream.resume();
+})
+// {/fact}
+
+// {fact rule=path-traversal@v1.0 defects=1}
+///unzipper lib
+fs.createReadStream('./bad.tar').pipe(extract);
+const fs = require('fs');
```

Biome (2.1.2)

[error] Shouldn't redeclare 'fs'. Consider to delete it or rename it.

'fs' is defined here:


(lint/suspicious/noRedeclare)